content/nginx-one/how-to/config-sync-groups/add-file-csg.md

```diff
@@ -0,0 +1,70 @@
+---
+docs: 
+doctypes:
+    - task
+tags:
+    - docs
+title: Add a file to a Config Sync Group
+toc: true
+weight: 400
```

Comment:
```suggestion
title: Add a file to a Config Sync Group
weight: 400
toc: true
type: how-to
product: NGINX One
docs: DOCS-000
```

Reply:
Does this matter -- in https://github.com/nginx/documentation/pull/90/files#diff-dfc68b874162469152f3646287019eae3428c3afb8bc5a75bb5becb5fe64b11cR1, I learned that some linters set up this indentation.

Reply:
Yes, but I think that was merged before I had a chance to make an edit suggestion.

It's outside of scope for this PR: we can't impose linting rules on people until we have provided them an alternative.

It's a consistency/style concern, but not a structural engineering one that should block PRs, IMO.